code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -515,6 +515,14 @@
 	cost = 50
 	excludefrom = list(UPLINK_TYPE_NUCLEAR, UPLINK_TYPE_SST)
 
+/datum/uplink_item/device_tools/hyper_medipen
+	name = "Hyperregenerative Medipen"
+	desc = "Autoinjector filled with a variety of medical chemicals. It rapidly heals conventional injuries and genetic damage, but loses potency just as quickly. May have side effects if used in succession."
+	reference = "HMP"
+	item = /obj/item/reagent_containers/hypospray/autoinjector/hyper_medipen
+	cost = 10
```

Comment:
More of a balance team related thing but this seems like a good price due to the nature of this pen's chemicals. This could also just be made into a maintenance loot item, but they'll have to decide on that.

Reply:
This is not a review of anything, just a personal opinion about price. I am resolving this conversation. 
Any opinions about the design of a PR or anything of that nature should be posted as a regular comment.
Thanks!